src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentService.kt

```diff
@@ -82,9 +83,9 @@ class IncidentService(
         this.agency = agency
         this.questionnaire = questionnaire
         this.reportingStaff = reportedStaff.staff
-        this.reportedDate = request.reportedDateTime
+        this.reportedDate = request.reportedDateTime.truncatedTo(ChronoUnit.DAYS)
```

Comment:
clearing the time portion from reportedDate and incidentDate mirrors how it is stored when creating incidents in Nomis